components/Caretaker.js

```diff
@@ -3,6 +3,8 @@ import { Button, Header } from 'react-native-elements';
 import { View, StyleSheet, FlatList, Modal } from "react-native";
 import Times from './Times';
 import AddTask from './AddTask';
+import normalize from 'react-native-normalize';
```

Comment:
Does this need to import on all files?  And if so, can we normalize on main? Or would it be easier for now to keep normalizing on each file? I'm down for whatever's easiest!

Reply:
I'm not sure but I think it would be a good idea to look into this as an extension. Every example I used was one file. The `normalize` keyword has to be used on the styling properties separately, though I'm not sure that means it needs to be imported more than once.